src/nodeinfoprovider/nodeinfoprovider.cpp

```diff
@@ -42,6 +42,7 @@ Error NodeInfoProvider::Init()
         return AOS_ERROR_WRAP(ret);
     }
 
+    mNodeInfo.mName     = "zephyr";
```

Comment:
Add AOS_NODE_NAME option to Kconfig and use it

Reply:
fixed